deepfence_frontend/apps/dashboard/src/features/settings/pages/ChangePassword.tsx

```diff
@@ -0,0 +1,141 @@
+import { Link, useFetcher } from 'react-router-dom';
+import { ActionFunction, redirect } from 'react-router-dom';
+import { Button, Card, TextInput, Typography } from 'ui-components';
+
+import { getUserApiClient } from '@/api/api';
+import { ApiDocsBadRequestResponse } from '@/api/generated';
+import { SettingsTab } from '@/features/settings/components/SettingsTab';
+import { ApiError, makeRequest } from '@/utils/api';
+
+export type changePasswordActionReturnType = {
+  error?: string;
+  fieldErrors?: {
+    old_password?: string;
+    new_password?: string;
+    confirm_password?: string;
+  };
+};
+
+export const changePasswordAction: ActionFunction = async ({
```

Comment:
we can name it `action` since there is only one action in this page, also it does not need to be exported individually.

Reply:
change to action